src/main/java/com/leverx/springapp/demo/repository/UserJdbcRepositoryImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.leverx.springapp.demo.repository;
+
+import com.leverx.springapp.demo.model.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+@RequiredArgsConstructor
+public class UserJdbcRepositoryImpl implements UserJdbcRepository {
+    private final JdbcTemplate jdbcTemplate;
+
+    public static final String INSERT_INTO_USERS_FIRST_NAME_SECOND_NAME_VALUES = "INSERT INTO users (first_name, second_name) values (?,?)";
```

Comment:
static final vars must be before final vars. Why checkstyle doesn't say anything?